rplugin/python3/deoplete/sources/tabnine.py

```diff
@@ -114,12 +114,22 @@ def get_tabnine_path(binary_dir):
     SYSTEM_MAPPING = {
         'Darwin': 'apple-darwin',
         'Linux': 'unknown-linux-gnu',
+        'Windows': 'pc-windows-gnu'
     }
     versions = os.listdir(binary_dir)
     versions.sort(key=parse_semver, reverse=True)
     for version in versions:
-        triple = '{}-{}'.format(platform.machine(),
+        triple = '{}-{}'.format(parse_architecture(platform.machine()),
                                 SYSTEM_MAPPING[platform.system()])
-        path = os.path.join(binary_dir, version, triple, 'TabNine')
+        if platform.system() == 'Windows':
+            path = os.path.join(binary_dir, version, triple, 'TabNine.exe')
+        else:
+            path = os.path.join(binary_dir, version, triple, 'TabNine')
```

Comment:
This would be somewhat nicer with an additional variable for the suffix, either ‘’ or ‘.exe’. Then the os.path.join code wouldn’t need to be duplicated.